tests/conftest.py

```diff
@@ -133,6 +133,31 @@ def clean(dsw_sdk):
     _clean(dsw_sdk)
 
 
+# ----------------------------- Questionnaires -------------------------------
+
+
+@pytest.fixture
+def qtn_data(request):
+    data = {
+        'name': 'My questionnaire',
+        'packageId': 'dsw:root:2.3.0',
+        'sharing': 'RestrictedQuestionnaire',
+        'visibility': 'PrivateQuestionnaire',
+        'tag_uuids': [],
+        'template_id': None,
+    }
+    return get_data(request, 'qtn_data_data', data)
+
+
+@pytest.fixture
+def qtn_tpl_data(request):
+    data = {
+        'name': 'My questionnaire',
+        'questionnaireUuid': 'a04d83ab-9553-4862-967d-54cf37579c44',
```

Comment:
This questionnaire also doesn't exist on fresh DSW instnace – same problem as above.